src/Core/x/xCutscene.h

```diff
@@ -92,11 +92,13 @@ struct xCutscene
 };
 
 extern uint32 gFrameCount;
+extern xCutscene sActiveCutscene;
```

Comment:
All variables should be defined/declared in their C++ files, not header files

Reply:
So functions/structs in header files, variables in c++ files?

Reply:
Yes. Function declarations in header files, function implementations in C++ files, variable declarations in C++ files is the pattern we have stuck to  so far